src/modules/About/Cards/Card.tsx

```diff
@@ -20,6 +20,10 @@ const Container = styled.div`
   align-items: center;
   justify-content: space-around;
   text-align: center;
+  @media (max-width: 768px) {
```

Comment:
the Card component isn't used anywhere, please remove it

Reply:
It used at /Users/hilashay/Development/shopping-app/src/modules/About/Cards/Cards.tsx

Reply:
I think you meant to the card that was at the review section